pkg/client/client.go

```diff
@@ -460,7 +461,10 @@ func (c *Client) Fetch(ctx context.Context, request FetchRequest) (res *FetchRes
 			)
 			for {
 				resp, err := stream.Recv()
-				if err == io.EOF {
+				st, ok := status.FromError(err)
+
+				if (ok && st.Code() == gcodes.Canceled) || err == io.EOF {
```

Comment:
if `err` is nil, `ok` will be `true` and `st` will be `nil`, so panic... needs a better `if`

Reply:
Fixed